scripts/gm/bin/gm

```diff
@@ -2,17 +2,16 @@
 
 set -eu
 
-# Load lib-gm either from the local folder or from the global $HOME/.gm/bin/lib-gm or finally from an absolute path set
-# as an environment variable
+# Load lib-gm either from the local folder or from the global $HOME/.gm/bin/lib-gm
 SCRIPT_0="${0:-$HOME/.gm/bin/gm}"
 export SCRIPT_DIR="${SCRIPT_0%%gm}"
 export LOCAL_LIB_GM="${SCRIPT_DIR}lib-gm"
 if [ -f "$LOCAL_LIB_GM" ]; then
   . "$LOCAL_LIB_GM"
 elif [ -f "$HOME/.gm/bin/lib-gm" ]; then
   . "$HOME/.gm/bin/lib-gm"
-elif [ -f "$LIB_GM_PATH/lib-gm" ]; then
-  . "$LIB_GM_PATH/lib-gm"
+elif [ -f "${LIB_GM-:}" ]; then
```

Comment:
All right, so it's going to be `LIB_GM`. (As I said, I don't mind either way, use whatever is best for your use-case.)

One last small suggestion is a typo fix: The `:` comes before the `-` for an empty default value.

Like this:
```suggestion
elif [ -f "${LIB_GM:-}" ]; then
```

Feel free to merge it from the web interface, I think it allows you to do it for cross-repo PRs too. (Or fix it on your side and push it. Again, up to you.)